.github/actions/start-local-server/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
         # Give it some time before starting to poll.
         sleep 10
         started=1
-        for i in `seq 1 120`;
+        for i in `seq 1 240`;
```

Comment:
I had to bump this out in my testing; I suspect it's taking longer than usual here due to cache misses. The cache should be populated on merge to main.